src/Symfony/Component/Form/Resources/translations/validators.el.xlf

```diff
@@ -14,6 +14,126 @@
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Please enter a valid date and time.</source>
+                <target>Please enter a valid date and time.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>Παρακελούμε εισάγετε μία έγκυρη ημερομηνία.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>Παρακαλούμε επιλέξτε ένα έγκυρο αρχείο.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>Το κρυφό πεδίο δέν είναι έγκυρο.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Παρακαλούμε εισάγετε έναν ακέραιο αριθμό.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Παρακαλούμε επιλέξτε μία έγκυρη γλώσσα.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Παρακαλούμε επιλέξτε μία έγκυρη τοπικοποίηση.</target>
+            </trans-unit>
+            <trans-unit id="114">
+                <source>Please enter a valid money amount.</source>
+                <target>Παρακαλούμε εισάγετε ένα έγκυρο χρηματικό ποσό.</target>
+            </trans-unit>
+            <trans-unit id="115">
+                <source>Please enter a number.</source>
+                <target>Παρακαλούμε εισάγετε έναν αριθμό.</target>
+            </trans-unit>
+            <trans-unit id="116">
+                <source>The password is invalid.</source>
+                <target>Ο κωδικός δέν είναι έγκυρος.</target>
+            </trans-unit>
+            <trans-unit id="117">
+                <source>Please enter a percentage value.</source>
+                <target>Παρακαλούμε εισάγετε μία ποσοστιαία τιμή.</target>
+            </trans-unit>
+            <trans-unit id="118">
+                <source>The values do not match.</source>
+                <target>Οι τιμές δέν ταιριάζουν.</target>
+            </trans-unit>
+            <trans-unit id="119">
+                <source>Please enter a valid time.</source>
+                <target>Παρακαλούμε εισάγετε μία έγκυρη ώρα.</target>
+            </trans-unit>
+            <trans-unit id="120">
+                <source>Please select a valid timezone.</source>
+                <target>Παρακαλούμε επιλέξτε μία έγυρη ζώνη ώρας.</target>
+            </trans-unit>
+            <trans-unit id="121">
+                <source>Please enter a valid URL.</source>
+                <target>Παρακαλούμε εισάγετε ένα έγυρο σύνδεσμο.</target>
```

Comment:
Does this say "URL" or "link"? It is technically a difference. Please make sure it says URL.